copy_this/application/commands/cacheclearcommand.php

```diff
@@ -48,12 +48,20 @@ public function help(oxIOutput $oOutput)
     public function execute(oxIOutput $oOutput)
     {
         $oInput = $this->getInput();
+
+        $oOutput->writeLn('Clearing OXID db and backend cache..');
+        /** @var oxCache $oCache */
+        $oCache = oxNew('oxcache');
```

Comment:
i am not sure about a separate command, i would prefer arguments for that cache clear command if you only want to flush a specific cache. about oxcache i thinking about simply checking with class_exists